Gem/Settings/Scenes/SettingsScene.swift

```diff
@@ -36,6 +36,8 @@ struct SettingsScene: View {
                     ListItemView(title: Localized.Settings.security, subtitle: .none, image: Image(.settingsSecurity))
                 }
             }
+            .id(TabScrollToTopId.settings)
```

Comment:
is it possible to setup `.id()` from the outside of SettingsScene? in navigation coordinator? to avoid having too much code inside each screen

Reply:
removed since, it was request to remove it by now